apps/docs/src/pages/index.module.css

```
@@ -37,3 +40,146 @@
+  display: grid;
+  gap: 1.75rem;
+  max-width: 1080px;
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.featureEyebrow {
+  display: inline-flex;
+  margin-bottom: 0.5rem;
+
+  color: var(--ifm-color-primary);
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)